napari/utils/naming.py

```diff
@@ -75,3 +75,10 @@ def magic_name(value, *, path_prefix):
     for name in names:
         if name.isidentifier() and name in varmap and varmap[name] is value:
             return name
+
+
+def force_name_unique(name, names):
+    if names.count(name) == 0:
```

Comment:
maybe this is nitpicky but it would be slightly more performant if you used

```suggestion
    if name not in names:
```

since in the case where it's `False` it will stop at the first duplicate name instead of continuing to search the rest of the list. i don't think it'll be noticeable unless you have like 1000 layers tho

Reply:
Will change, I actually had a version where i needed count > 1 which is why it was there. Good catch